pkgs/tools/misc/uhubctl/default.nix

```
@@ -3,21 +3,20 @@
 , libusb
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "uhubctl";
-  version = "unstable-2019-07-31";
+  version = "2.1.0";
 
   src = fetchFromGitHub {
     owner = "mvp";
     repo = "uhubctl";
-    rev = "1961aa02e9924a54a6219d16c61a0beb0d626e46";
-    sha256 = "15mvqp1xh079nqp0mynh3l1wmw4maa320pn4jr8bz7nh3knmk0n1";
+    rev = "refs/tags/v${version}";
```

shouldn't need tags prefix
```suggestion
    rev = "v${version}";
```